frontend/src/components/profile/settings.js

```diff
@@ -10,7 +10,8 @@ import {
   MenuItem,
   Button,
   Switch,
-  Checkbox
+  Checkbox,
+  localStorage
```

Comment:
on line 73 you have used local storage

Reply:
@Anshgrover23, localStorage is a built-in Javascript library, and it shouldn't be imported, especially because it's imported as a Material-UI dependency, which is completely wrong.